functions-methods.js

```diff
@@ -19,9 +32,34 @@
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
+function typeOfEmail(arr) {
+    let mailArray = Array.from(arr);
+    let counter = 0;
+    let domain = "";
+    let type = "";
+    for (let i = 0; i < mailArray.length; i++) {
+        if (mailArray[i] !== "@") {
+            counter++;
+        } else {
+            domain = arr.slice(counter + 1);
+            break;
+        }
+    }
+    console.log(domain)
+    switch (domain) {
+        case "novi-education.nl":
+            type = "Student";
+            break;
+        case "novi.nl":
+            type = "Medewerker";
+            break;
+        case "outlook.com":
+            type = "Extern";
+    }
+    return type
```

Comment:
Je zou zelfs meteen de waarde terug kunnen geven
```suggestion
    switch (domain) {
        case "novi-education.nl":
            return "Student";
        case "novi.nl":
            return "Medewerker";
        case "outlook.com":
            return "Extern";
    }
```